### PR TITLE
LITE-29414 Library common utilities for eaas extensions

### PR DESCRIPTION
**Pull request by jonatrios**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        - '*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```
Comment (maxipavlovic) [inline .github/workflows/build.yml]: 3.7 is not needed 100%, maybe we can drop more.

File: .github/workflows/deploy.yml
Comment (maxipavlovic) [inline .github/workflows/deploy.yml:1]: I see a lot of build/deploy tasks in PR which seem to be excessive. Generally we check on PR and deploy on tag.

**Review by maxipavlovic — APPROVED** [review]
In general, a great job!

File: .github/workflows/build.yml (hunk 1 above)
Comment (jonatrios) [reply .github/workflows/build.yml]: Perfect will drop 3.7 and 3.8

**Pull request merged**